ribbon/sign.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#----------------------------------------------------------------------------
+# Created By: Steven (steven@ribbon.finance)
+# Created Date: 04/04/2022
+# version ='0.01'
+# ---------------------------------------------------------------------------
+''' Module to generate signature to bid '''
+# ---------------------------------------------------------------------------
+
+# ---------------------------------------------------------------------------
+# Imports
+# ---------------------------------------------------------------------------
+import eth_keys
+from encode import TypedDataEncoder
+
+# ---------------------------------------------------------------------------
+# Signature Generator
+# ---------------------------------------------------------------------------
+class SignatureGenerator:
```

Comment:
Like above, we can call this just `Signature`,
as we know we'll get one through this object.

Reply:
Okay